docker-compose.test.yml

```diff
@@ -1,11 +1,50 @@
 version: '3'
 services:
+  audit:
+    image: node:10
+    container_name: test_audit
+    ports:
+      - "${AUDIT_PORT:-3004}:3004"
+    working_dir: "/src"
+    command: ["bash", "-c", "yarn && yarn run migrate run &&yarn run start:dev"]
+    environment:
+      CONFIG_PATH: ./tests/config.json
+    volumes:
+      - ./:/src/:z
+      - ./config/:/etc/reviewer/:z
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://localhost:3004/health" ]
+      interval: 2s
+      timeout: 30s
+      retries: 10
+    depends_on:
+      - rabbitmq
+      - postgres
+
+  rabbitmq:
+    image: rabbitmq:3-management
+    container_name: test_rabbitmq
+    ports:
+      - "15672:15672"
+      - "5672:5672"
+    healthcheck:
+      test: [ "CMD", "rabbitmqctl", "status" ]
+      interval: 2s
+      timeout: 30s
+      retries: 10
+
   postgres:
     image: postgres:11
+    container_name: test_postgres
     ports:
-       - "5432:5432"
+      - "5432:5432"
     volumes:
-       - pg-data:/var/lib/postgresql/data
+      - pg-data:/var/lib/postgresql/data
```

Comment:
are you sure you want persistence? May make testing simpler to discard data on `stop`

Reply:
fixed